keyboards/neson_design/nico/info.json

```
@@ -0,0 +1,123 @@
+{
+    "manufacturer": "Neson Design",
+    "keyboard_name": "Nico",
+    "maintainer": "yulei",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "indicators": {
+        "caps_lock": "E2"
+    },
+    "matrix_pins": {
+        "cols": ["C7", "C6", "B6", "B5", "B4", "D7", "D6", "D4", "D5", "D3", "D2", "B7", "B3", "B2", "B1", "F6", "F7"],
+        "rows": ["F0", "E6", "F5", "F4", "F1"]
+    },
+    "processor": "atmega32u4",
+    "rgblight": {
+        "brightness_steps": 8,
+        "hue_steps": 8,
+        "led_count": 5,
+        "pin": "B0",
+        "saturation_steps": 8
+    },
+    "url": "",
+    "usb": {
+        "device_version": "0.0.1",
+        "no_startup_check": true,
+        "pid": "0x6E65",
+        "vid": "0x4E65"
+    },
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                { "label": "K017", "matrix": [0, 15], "w": 1, "x": 0, "y": 0 },
```

```suggestion
                { "label": "K017", "matrix": [0, 15], "x": 0, "y": 0 },
```
etc...
`w: 1` is the default.

---

The rest of these need to be done.